UM/View/GL/OpenGLContext.py

```diff
@@ -167,7 +168,7 @@ def detectBestOpenGLVersion(cls, force_compatability: bool) -> Tuple[Optional[in
                     Logger.log("e", "Could not initialize OpenGL to get gpu type")
                 else:
                     # WORKAROUND: Cura/#1117 Cura-packaging/12
```

Comment:
Is this the case with PyQt6?

Reply:
@rburema this question looks like your cup-of-tea.

Reply:
I think this could be something Qt just passes from a driver it assumes is working correctly to us. With all the buggy graphics cards drivers out there I'm not comfortable removing this.

Reply:
Agreed